core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -90,6 +98,18 @@ protected void load() {
     activeTransactionManagementExpirationTimeMillis =
         getLong(getProperties(), ACTIVE_TRANSACTION_MANAGEMENT_EXPIRATION_TIME_MILLIS, -1);
     defaultNamespaceName = getString(getProperties(), DEFAULT_NAMESPACE_NAME, null);
+    crossPartitionScanEnabled = getBoolean(getProperties(), CROSS_PARTITION_SCAN, false);
```

Comment:
The default value is `false` in v4.0.0 or later, but it should be `true` in v3.x for backward compatibility and warned when using it with `SERIALIZABLE`. I will handle this after this PR is merged to the master.